documentation/modules/proc-standard-auth-service-ha.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// assembly-deploying-auth-services.adoc
+
+[id='proc-deploying-standard-auth-service-{context}']
+= Deploying the `standard` authentication service for HA
+
+To implement the `standard` authentication service, you deploy it.
```

Comment:
Perhaps include a brief sentence describing why the user should perform this procedure?